pkg/util/clusteragent/cluster_agent.auth_token

```diff
@@ -0,0 +1 @@
+27a45541147ba5d91a76c1b22a105e475312a8de6166196acade42067155e472
```

Comment:
I don't think you meant to add this file to the PR. Where does it come from btw?

Reply:
probably a file not cleaned up after unit test 🤔